src/app/panels/file-panel.js

```diff
@@ -136,24 +133,20 @@ function filepanel (localRegistry) {
   configExplorer.ensureRoot()
   var websocketconn = element.querySelector('.websocketconn')
   self._deps.fileProviders['localhost'].event.register('connecting', (event) => {
-    websocketconn.style.color = styles.colors.yellow
```

Comment:
so you are removing the color definintion of the icon for remixd - but what color does it change to?

Reply:
remixd will load as a module, so that UI is not anymore relevant (after merging to swap_it)